controller/controller.go

```diff
@@ -738,7 +749,9 @@ func main() {
 	if Ctrler.Leader {
 		kv.ValidateWebhookCert()
 		if isNewCluster && *noDefAdmin {
-			clusHelper.DeleteUser(common.DefaultAdminUser)
+			if err := clusHelper.DeleteUser(common.DefaultAdminUser); err != nil {
+				log.WithFields(log.Fields{"error": err}).Fatal("DeleteUser")
+			}
```

Comment:
Looks neat.  In this case let's provide a more detailed message so if someone hit this, they would know what happened.

Reply:
changed.